### ISSUE TITLE
installation question

### ISSUE THREAD

**Issue by gsiaw** [ecd-plugin/ecd#29]
im getting an error when doing an installation of the plugin. both from manually entering the url and using the Drag and Drop installation on https://ecd-plugin.github.io/ecd/ page

```
Unable to read repository at https://ecd-plugin.github.io/update/content.xml.
sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

when i goto https://ecd-plugin.github.io/update/content.xml , i also dont get any xml file.  not sure if something is being blocked on my end. 

also this is eclipse Version: Neon.1 Release (4.6.1)

**Comment by RobertZenz** [comment]
That is most likely a duplicate of #11, please check your proxy configuration.

**Comment by pierreonthenet** [comment]
I have exactly the same problem with a fresh install of Eclipse Mars 2 64bit (Version: Mars.2 Release (4.5.2) Build id: 20160218-0600).
I don't have access to https://ecd-plugin.github.io/update/content.xml  as well, and this is NOT a proxy problem.
Does anyone can access this ? It seems that this plugin is unavailable...

**Comment by pbi-qfs** [comment]
The Update site at https://ecd-plugin.github.io/update/ is running fine, and most people do not have any problems installing the plugin.

As explained in #11, the error message of Eclipse is misleading - the "update.xml" is not needed anymore with modern eclipse versions, but it reports its missing as soon as anything is wrong with the network configuration.

As a workaround, you can manually download the feature and plugin files from https://ecd-plugin.github.io/update/ and place them in the correspondent folders in your eclipse installation.

**Comment by sadashiv07** [comment]
In Eclipse - Photon 
I'm getting an error when doing an installation of the plugin. both from manually entering the url and using the Drag and Drop installation on https://ecd-plugin.github.io/ecd/ page

Unable to read repository at https://ecd-plugin.github.io/update/content.xml.
sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
when I goto https://ecd-plugin.github.io/update/content.xml , i also dont get any xml file. not sure if something is being blocked on my end.
Some of the other plugins are working fine .

**Comment by pbi-qfs** [comment]
@sadashiv07 It must be something on your site, like a network scanner, which is blocking the HTTPS-connection to github.io. From other places, the access to the files is working without problems.

As a work-around, you can download the plugin as ZIP file from the releases tab of this site and then extract it to the "dropins" directory of your eclipse installation.
